doc/user/content/sql/create-index.md

```diff
@@ -170,7 +169,31 @@ are automatically created contain an index of all columns in the result set,
 unless they contain a unique key. (Remember that indexes store a copy of a
 row's indexed columns _and_ a copy of the entire row.)
 
+### Speed up filtering with indexes
+
+You can set up an index over a column were filtering by literal values or expressions are common to improve the performance.
+
+```sql
+CREATE MATERIALIZED VIEW active_customers AS
+    SELECT guid, geo_id, last_active_on
+    FROM customer_source
+    GROUP BY geo_id;
+
+CREATE INDEX active_customers_idx ON active_customers (guid);
+
+SELECT * FROM active_customers WHERE guid = 'd868a5bf-2430-461d-a665-40418b1125e7';
+
+-- Using expressions
+CREATE INDEX active_customers_exp_idx ON active_customers (upper(guid));
+
+SELECT * FROM active_customers WHERE upper(guid) = 'D868A5BF-2430-461D-A665-40418B1125E7';
```

Comment:
You may want to note that if your query has two predicates, like `WHERE upper(guid) = 'D868A5BF-2430-461D-A665-40418B1125E7' and geo_id = 'something'`, then the index you need is `ON active_customers(upper(guid), geo_id)`

Reply:
Added!